src/dotnet-format.csproj

```diff
@@ -19,7 +19,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
 
     <!-- Always run on the latest runtime installed. -->
-    <RuntimeFrameworkVersion>2.1.0-rc1</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion>3.1.0-preview3.19553.2</RuntimeFrameworkVersion>
```

Comment:
Targeting a preview will allow the rollforward mechanism to rollforward to previews.

Reply:
got it, so this is necessary for it to rollforward to 6.0 previews